ext/hathach/module.lb

```diff
@@ -109,19 +113,22 @@ def build(env):
             tusb_config["CFG_TUSB_MCU"] = "OPT_MCU_RP2040"
             env.copy("tinyusb/src/portable/raspberrypi/rp2040/", "portable/raspberrypi/rp2040/")
 
-    if env.has_module(":freertos"):
-        tusb_config["CFG_TUSB_OS"] = "OPT_OS_FREERTOS"
-        env.copy("tinyusb/src/osal/osal_freertos.h", "osal/osal_freertos.h")
-    else:
-        env.copy("tinyusb/src/osal/osal_none.h", "osal/osal_none.h")
+    # TinyUSB has a OS abstraction layer for FreeRTOS, but it is causes problems with modm
+    # if env.has_module(":freertos"):
+    #    tusb_config["CFG_TUSB_OS"] = "OPT_OS_FREERTOS"
+    #    env.copy("tinyusb/src/osal/osal_freertos.h", "osal/osal_freertos.h")
```

Comment:
```suggestion
    #    env.copy("tinyusb/src/osal/osal_freertos.h", "osal/osal_freertos.h")

    if env.has_module(":freertos"):
        env.log.warning("TinyUSB in modm is currently not using its FreeRTOS abstraction layer and is not thread-safe with FreeRTOS threads.")
```

Reply:
I would add a warning comment in the example source. If we have a specific tinyusb + FreeRTOS example and there is a tinyusb FreeRTOS integration I would assume that is what is being shown there.

Reply:
Oh yes, good idea!